client/luigi-element.d.ts

```diff
@@ -414,4 +414,27 @@ export interface LuigiClient {
   linkManager: () => LinkManager;
   uxManager: () => UxManager;
   publishEvent: (event: Event) => void;
+  /**
+   * Sets node parameters in Luigi Core. The parameters will be added to the URL.
+   * @param {Object} params
+   * @param {boolean} keepBrowserHistory
+   * @memberof LuigiClient
+   */
+  addNodeParams: (params: Object, keepBrowserHistory: boolean) => void;
+  /**
+   * Returns the node parameters of the active URL.
+   * Node parameters are defined like URL query parameters but with a specific prefix allowing Luigi to pass them to the micro frontend view. The default prefix is **~** and you can use it in the following way: `https://my.luigi.app/home/products?~sort=asc&~page=3`.
+   * <!-- add-attribute:class:warning -->
+   * > **NOTE:** some special characters (`<`, `>`, `"`, `'`, `/`) in node parameters are HTML-encoded.
+   * @param {boolean} shouldDesanitise defines whether the specially encoded characters should be desanitised
+   * @returns {Object} node parameters, where the object property name is the node parameter name without the prefix, and its value is the value of the node parameter. For example `{sort: 'asc', page: 3}`
+   * @memberof LuigiClient
+   */
+  getNodeParams: (shouldDesanitise: boolean) => Object;
```

Comment:
Again the same config as in Modal WC, when I try to get the node params inside a modal, I do not get the node params. But when the modal is opened in normal WC and not a modal it works well. 

Example:
```
import { LuigiElement } from "./luigi-element.js";
export default class extends LuigiElement {
  render(ctx) {

    return '<div>test</div>';
  }
  onContextUpdate(ctx) {
    console.log('context update', ctx);
  }

  afterInit(ctx) {
    // <------------------- getNodeParams() returns undefined --------------------> 
    console.log('context afterInit', ctx, this.LuigiClient.getNodeParams());
  }
}
```